.storybook/main.ts

```diff
@@ -27,4 +31,25 @@ module.exports = {
     check: true,
     checkOptions: {},
   },
+  babel: babelConfig,
+  webpackFinal: (config, { configType }) => {
```

Comment:
It looks like this is the preferred way to add custom `webpack` logic now: https://storybook.js.org/docs/react/configure/webpack.

Given that we removed most of the logic in #529, I made this change as well to conform to Storybook standards.